test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
The only change beyond the tuple change and the `.items()` change for this file is adding a line of whitespace on L51 and deleting a line of whitespace on L374. Could you point me to what spurious changes in this file you want to see reverted?